room/models.py

```diff
@@ -1,7 +1,7 @@
 from django.db import models
 
 
-class Room(models.Model):
+class RoomModel(models.Model):
```

Comment:
Important to differentiate between Django models and Domain entities. Hence the explicit "Model" suffix.